general/package/ingenic-osdrv-t30/files/script/load_ingenic

```diff
@@ -78,8 +78,17 @@ if [ $? -ne 0 ]; then
     check_return "insmod audio"
 fi
 
-# Need for claim gpio
-insmod ${KMOD_PATH/%\//}/gpio.ko
-echo $(yaml-cli -i /etc/majestic.yaml -g .nightMode.irCutPin1) >/proc/jz/claim/gpio
-echo $(yaml-cli -i /etc/majestic.yaml -g .nightMode.irCutPin2) >/proc/jz/claim/gpio
-# echo $(yaml-cli -i /etc/majestic.yaml -g .nightMode.irSensorPin) >/proc/jz/claim/gpio
+lsmod | grep "gpio" >/dev/null
+if [ $? -ne 0 ]; then
+    insmod ${KMOD_PATH/%\//}/gpio.ko
+    check_return "insmod gpio"
+fi
+
+if [ ! -z "$(lsmod | grep "gpio")" ]; then
+	GPIOS=".nightMode.irCutPin1 .nightMode.irCutPin2 .nightMode.irSensorPin .nightMode.backlightPin .audio.speakerPin"
+	for GPIO in $(echo ${GPIOS} | tr ' ' '\n')
```

Comment:
Whitespace is a delimiter is ash, no need to replace it with line feed.

Reply:
Thanks, it will be fixed in the next update.